microsite/blog/2023-07-08-stash-adopter-post.mdx

```diff
@@ -0,0 +1,185 @@
+
+To consolidate application health data, simplify support and clarify ownership for those on development teams as well as for those in non-engineering roles, we implemented an instance of Backstage as an internal service catalog we call Elevate.
+
+Utilizing Backstage as a framework, Elevate aims to improve quality and domain expertise for all things code and process at Stash.. The key to accomplishing this goal is to leverage the processes and integrations we already use at Stash, but centralize and surface the data in a way that is digestible for engineers and non-engineers alike.
+
+For the benefit of a self-managed internal service catalog to outweigh the change in processes and the cost of the catalog's infrastructure, it had to be easy to adopt and have an enlightening effect on teams.
+
+
+## Ownership visibility
+
+Elevate acts as a centralized hub for registering services, documenting ownership information, and establishing clear accountability. It provides a holistic view of distributed systems and the shared modules and libraries they use by making it easier to track ownership and ensure effective communication and collaboration.
+
+The Backstage [Software Catalog](https://backstage.io/docs/features/software-catalog/) model allows service-level tracking, as well as dashboards for Groups (teams or squads), Systems (business units or focus areas), Domains (organization units) or even company wide.
+
+
+### Easy adoption and catalog usage
+
+The Backstage open source plugins provide the nuts and bolts for our internal service catalog functionality. Backstage's community plugins assist with the basics of catalog management. They also allow for dynamically changing the type content based on which entry is being viewed in the catalog (library, service, API, team, etc.). Each view can be custom-tailored with the information for that type of entry, which allows for shared libraries to have an overview page focused on getting engineers to documentation; while the overview page for a deployed service can focus on the current incident status and on-call information as the most important thing. The plugin for searching allows for easy discovery of items or documentation throughout the catalog.
+
+
+###### Example catalog-info.yaml file to add service to Elevate
+
+
+### Connecting to subject matter experts
+
+Elevate leverages the CODEOWNERS integration to establish expertise of specific codebases within services and ensure accountability and clarity within the development teams. Codespace ownership is typically defined with required reviewers by code owners and admins that provide additional support, but the CODEOWNERS plugin allows for clarity into who the  subject matter expert of the codebase is, not just the administrative owner. The plugin also displays recent changes that have been merged into the codebase, including the authors and approvers of the changes and other valuable metadata.
```

Comment:
i'll be honest, i'm not familiar with the codeonwers plugin.  as a reader to the blog, this sort of infers i can go to the plugins list and find it.  is this perhaps a more custom implementation of a catalog ingestion for codeonwers as opposed to a true backstage plugin?

the context is great; useful to hear Elevate's balance of entity ownership vs. code owner, but as an integrator that wouldn't be clear how i'd do the same.